sql/hive-thriftserver/v2.3.5/src/main/scala/org/apache/spark/sql/hive/thriftserver/ThriftserverShimUtils.scala

```diff
@@ -17,6 +17,11 @@
 
 package org.apache.spark.sql.hive.thriftserver
```

Comment:
During JDK11 testing and review, we has been skipped renaming in order to focus JDK11 related stuff by minimizing PR diff. We may need to rename this file directory `v2.3.5` to `v2.3.6` again for consistency later. If the test pass, I'd like to merge this AS-IS PR first.

cc @gatorsmile , @srowen